src/main/java/com/google/cloudassets/discovery/AssetJsonParser.java

```diff
@@ -0,0 +1,66 @@
+package com.google.cloudassets.discovery;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The AssetJsonParser class maps a json node into a list of Maps in which each map represents
+ * a different asset object data.
+ */
+public class AssetJsonParser {
```

Comment:
I don't understand the code in this file does...
Can you improve the doc, explain the response to which type of requests this parses, or even include a (possibly shortened) example of the input and output?

Reply:
Added more detailed documentation for this class - please let me know if it is still not clear enough